### PR TITLE
Add http 3xx status codes

### PR DESCRIPTION
**Pull request by hugglesfox**
Add 301, 302 and 303 status codes to `http_status_code`.

I didn't bother adding them to `code_to_status` as the default `static_cast` should handle it.

### CODE REVIEW

**Review by macite — CHANGES_REQUESTED** [review]
Perhaps we look at removing `code_to_status`... what do you think? This isn't had public so it could just be a static cast.

Lets fix that as part of this PR.

**Comment by hugglesfox** [comment]
Sounds like a good idea to me. I'll do that now.

File: coresdk/src/backend/web_driver.cpp [hunk 1]
```diff
@@ -173,7 +151,7 @@ namespace splashkit_lib
 
         long status;
         curl_easy_getinfo(curl_handle, CURLINFO_RESPONSE_CODE, &status);
-        result->code = code_to_status((int)status);
+        result->code = static_cast<http_status_code>((int)status);
```
Comment (macite) [inline coresdk/src/backend/web_driver.cpp:154]: Remove the (int) cast as well - the static cast should be fine without that.

**Review by macite — CHANGES_REQUESTED** [review]
Just a simple fix - should be able to merge then. Thanks for the contributions.

**Review by macite — APPROVED** [review]
All good!

**Pull request merged**